src/cmd/flux-job.c

```diff
@@ -1214,13 +1223,14 @@ int cmd_eventlog (optparse_t *p, int argc, char **argv)
     }
 
     ctx.id = parse_arg_unsigned (argv[optindex++], "jobid");
+    optparse_getopt (p, "path", &ctx.path);
```

Comment:
I see your point.  It might be better.  Let me see how it works out.

Reply:
If you want to defer that and get this in ASAP since you're dealing with personal stuff today, feel free to set merge-when-passing and submit any cleanup in another PR...